server/game/PlayActions/MarshalCardAction.js

```diff
@@ -4,6 +4,7 @@ const Costs = require('../costs');
 class MarshalCardAction extends BaseAbility {
     constructor() {
         super({
+            abilitySourceType: 'game',
```

Comment:
I'd be open to alternate names for this property. It's meant to distinguish between 'game abilities' and 'card abilities.' The name `abilityType` is already used to distinguish between the various ability timings (interrupt, reaction, forced interrupt, etc). Perhaps that itself should be renamed, but even so it isn't very descriptive for this situation. RRG doesn't really offer any guidance on language here either (only one mention of "game ability" and it's in reference to duplicates).